Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/Rendering/BocColumnRenderer.cs

```diff
@@ -129,10 +129,14 @@ public void RenderDataColumnDeclaration (BocListRenderingContext renderingContex
         int rowIndex,
         string? cellID,
         IReadOnlyCollection<string> headerIDs,
+        bool[] columnsWithValidationFailures,
```

Comment:
We used IReadOnlyCollection instead of array because of the immutatility. I think using IReadOnlyList<bool> would be more consistent?

(everything in me screams to use BitArray, but that's not really "better" of course)